data/sql/001_CREATE_USERS_TABLE.sql

```diff
@@ -2,8 +2,7 @@
 CREATE TABLE IF NOT EXISTS `Users` (
     `id` INT NOT NULL AUTO_INCREMENT,
     `email` VARCHAR(100) NOT NULL,
-    `username` VARCHAR(30) 
-    NOT NULL UNIQUE DEFAULT (substring_index(email, '@', 1)),
+    `username` VARCHAR(30) NOT NULL UNIQUE DEFAULT (substring_index(email, '@', 1)),
```

Comment:
`(substring_index(email, '@', 1))` isn't needed since we allow users to create their own usernames. I think I may have suggested to you before this without realizing

Reply:
Resolved.